praw/models/inbox.py

```diff
@@ -25,6 +26,47 @@ def all(self, **generator_kwargs):
         return ListingGenerator(self._reddit, API_PATH['inbox'],
                                 **generator_kwargs)
 
+    def collapse(self, items):
+        """Mark an inbox message as collapsed.
+
+        :param items: A list containing instances of :class:`.Message` or
+            list of t4 message fullnames.
+
+        Requests are batched at 25 items (reddit limit).
+
+        For example, to collapse all unread Messages, try:
+
+        .. code:: python
+
+            from praw.models import Message
+            unread_messages = []
+            for item in reddit.inbox.unread(limit=None):
+                if isinstance(item, Message):
+                    unread_messages.append(item)
+            reddit.inbox.collapse(unread_messages)
+
+        To collapse a list of modmail messages by fullnames, try:
+
+        .. code:: python
+
+            messages = []
+            for message in reddit.subreddit('mod').mod.inbox(limit=None):
+                messages.append(item.fullname)
+            reddit.inbox.collapse(messages)
+
+        .. seealso::
+
+           :meth:`.Message.uncollapse`
+
+        """
+        while items:
+            if isinstance(items[0], Message):
+                data = {'id': ','.join(x.fullname for x in items[:25])}
```

Comment:
I'll just remove the extra feature thing. Didn't realize adding in additional support for something that could be useful would require modifying the `mark_(un)read` and similar. I'll adjust their sort order as well and will have that tonight.